user_manual/groupware/sync_thunderbird.rst

```diff
@@ -22,6 +22,8 @@ Contacts
 
 If you later want to add a new address book, you can redo all of those steps and only the books not already synchronized will be suggested.
 
+.. note:: If your account uses two-factor authentication, you need a `dedicated app password <../session_management.html#managing-devices>`_ for login rather than your
```

Comment:
Could you please link to the anchor https://github.com/nextcloud/documentation/blob/6bc5d2ce34d0813ed10810a01b97715949c9b029/user_manual/session_management.rst#L19? This will keep the link working even if the page/section moves. 

https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#use-a-custom-anchor